azext_edge/edge/_help.py

```diff
@@ -443,6 +443,22 @@ def load_iotops_help():
 
                       After the foundation layer has been installed the `az iot ops create` command should
                       be used to deploy an instance.
+
+                      Note: --*-config options allow override of default config settings.
```

Comment:
--*-

Reply:
The asterisk meant to be wildcard such as `--acs-config` , `--ssc-config`